README.md

```diff
@@ -34,4 +34,15 @@ https://pypi.org/project/psycopg2/
 
 ``` pip install coreapi```
 
-https://www.coreapi.org
\ No newline at end of file
+https://www.coreapi.org
+
+5. django-phonenumber-field
+
+``` pip install django-phonenumber-field[phonenumbers]```
```

Comment:
is there a one command line to install everything for python backend dependencies?